salesagility/SuiteCRM
# Clean up a bunch of unit tests.

## Description
This removes a bunch of dead variables, unnecessary blank lines,
poor formatting, etc.

I tried my best not to mess with the contents of any tests that were
marked incomplete.

## Motivation and Context
Just code cleanup.

## How To Test This
Make sure the tests continue to pass, make sure the PR doesn't remove anything important.

## Types of changes

### Final checklist
